### ISSUE TITLE
New Training Data

### ISSUE THREAD

**Issue by declan727** [jomjol/AI-on-the-edge-device#3505]
### Files

[learn.tar.gz](https://github.com/user-attachments/files/18469602/learn.tar.gz)

it is working great except for I think it is having an issue with the number 0, 1, 6 all the others I believe read translate just fine. I have tried many settings and the different models and dig-cont_0700_s3_q has been the best in my setup the water meter I have is a Neptune ProCoder T-10 Meter.

This project is amazing.

Thank you

**Comment by caco3** [comment]
Not sure what you did, but there are some numbers clearly labled wrong, eg.

`2.9_main_dig5_20250118-100429.jpg`

![Image](https://github.com/user-attachments/assets/74343805-d0c3-4353-8173-9cf1be7e559f)

**Comment by declan727** [comment]
I do not know either I have not done anything custom other then trying different settings in the camera and trying different models.

**Comment by declan727** [comment]
I have gotten the collectdigits to work properly I will run it and fix the labeling and provide them to hopefully be included in the training model.

Thank you

**Comment by declan727** [comment]
[labeled.tar.gz](https://github.com/user-attachments/files/18548353/labeled.tar.gz)

is there anything I can do to help? I did not think I would be able to do the training on my own.

**Comment by jomjol** [comment]
> [labeled.tar.gz](https://github.com/user-attachments/files/18548353/labeled.tar.gz)
> 
> is there anything I can do to help? I did not think I would be able to do the training on my own.

You need to label the numbers by hand and ensure, that they are consistent. Otherwise errors will be trained and implemented into the neural network. If the images look the same you can exclude them, because similar images do not help.
Also it would be good to have a full of examples from 0.0, 0.1, 0.2, .... 9.8, 9.9, that means more or less 100 different numbers. It is clear, that this takes a lot of time and effort, but only than a good recognition is ensured. 

Following example from your images, showing inconsitency:

![Image](https://github.com/user-attachments/assets/1850b870-fcb0-49ba-8348-d9a5e02c1f2b) 
both labeled with 6.9

or: 

![Image](https://github.com/user-attachments/assets/cc1b781e-1799-4d4e-b0f5-f47fdd754a38)
this is looking more like a 4.5 and not 5.5

**Comment by declan727** [comment]
ok I am sorry I thought that 6.9 was showing close to changing 6 to 7 and the 5.5 for 5 to 6 I am sorry if I misunderstood how the labeling was working. I got the the collectdigits executable to work from https://github.com/haverland/collectmeterdigits and it previewed the numbers and allowed me to tweak them. I will go over them again. Thank you for your patience and help.

**Comment by caco3** [comment]
Closing since there is no response since over a month
